CHANGELOG.md

```diff
@@ -1,4 +1,320 @@
-Please see the [Releases Page](https://github.com/kubernetes/kubernetes/releases)
+# Release Notes for Kubernetes 1.2.0
 
+## [Documentation](http://kubernetes.github.io) & [Examples](http://releases.k8s.io/release-1.2/examples)
+
+## Downloads
+
+binary | hash alg | hash
+------ | -------- | ----
+[kubernetes.tar.gz](https://storage.googleapis.com/kubernetes-release/release/v1.2.0/kubernetes.tar.gz) | md5  | `c0ce9e6150e9d7a19455db82f3318b4c`
+[kubernetes.tar.gz](https://storage.googleapis.com/kubernetes-release/release/v1.2.0/kubernetes.tar.gz) | sha1 | `52dd998e1191f464f581a9b87017d70ce0b058d9`
+
+## Changes since v1.1.1
+
+## Major Themes
+
+  * <strong>Significant scale improvements</strong>. Increased cluster scale by 400% to 1000 nodes with 30,000 pods per cluster.
+Kubelet supports 100 pods per node.
+  * <strong>Simplified application deployment and management. </strong>
+     * Dynamic Configuration (ConfigMap API in the core API group) enables application
+configuration to be stored as a Kubernetes API object and pulled dynamically on
+container startup, as an alternative to baking in command-line flags when a
+container is built. 
+     * Turnkey Deployments (Deployment API (Beta) in the Extensions API group)
+automate deployment and rolling updates of applications, specified
+declaratively. It handles versioning, multiple simultaneous rollouts,
+aggregating status across all pods, maintaining application availability, and
+rollback.
+  * <strong>Automated cluster management: </strong>
+     * Kubernetes clusters can now span zones within a cloud provider. Pods from a
+service will be automatically spread across zones, enabling applications to
+tolerate zone failure.
+     * Simplified way to run a container on every node (DaemonSet API (Beta) in the
+Extensions API group): Kubernetes can schedule a service (such as a logging
+agent) that runs one, and only one, pod per node.
+     * TLS and L7 support (Ingress API (Beta) in the Extensions API group): Kubernetes
+is now easier to integrate into custom networking environments by supporting
+TLS for secure communication and L7 http-based traffic routing.
+     * Graceful Node Shutdown (aka drain) - The new “kubectl drain” command gracefully
+evicts pods from nodes in preparation for disruptive operations like kernel
+upgrades or maintenance.
+     * Custom Metrics for Autoscaling (HorizontalPodAutoscaler API in the Autoscaling
+API group): The Horizontal Pod Autoscaling feature now supports custom metrics
+(Alpha), allowing you to specify application-level metrics and thresholds to
+trigger scaling up and down the number of pods in your application.
+  * <strong>New GUI</strong> (dashboard) allows you to get started quickly and enables the same
+functionality found in the CLI as a more approachable and discoverable way of
+interacting with the system. Note: the GUI is eanbled by default for new
+cluster creation, however, it does not 
```

Comment:
Who wrote that? I can't see the revision history on the doc and don't know what it was supposed to say.